docs/guide/dev/integration/react/README.md

```diff
@@ -215,6 +169,186 @@ component.editor.getData();
 
 Please note that this property is initialised asynchronously, after mounting the component.
 
+## Component Properties
+
+### `config`
+
+`Object`
+
+Custom configuration can be passed to the editor with the `config` property of the CKEditor 4 React component. The following example shows {@link features/toolbar/README how to change the contents of the toolbar}:
+
+```jsx
+<CKEditor
+	data="<p>Editor's content</p>"
+	config={ {
+		toolbar: [ [ 'Bold' ] ]
+	} }
+/>
+```
+
+{@linkapi CKEDITOR.config All configuration} options can be changed this way.
+
+Defaults to `{}`.
+
+### `data`
+
+`String`
+
+Data to be used to initially fill editor content. It's passed to {@linkapi CKEDITOR.editor#setData} method with no additional arguments.
+
+For example usage, look at [basic usage](#basic-usage) paragraph.
+
+Defaults to `''`.
+
+### `name`
+
+`String`
+
+If provided - became editor HTML element `id` and `name` properties.
+
+```jsx
+<CKEditor
+	name="myeditor"
+/>
+```
+
+Then in another place, it's possible to refer editor instance as:
+
+```js
+const editor = CKEDITOR.instances.myeditor;
+```
+
+Defaults to `undefined`.
+
+### `readOnly`
+
+`Boolean`
+
+Set the {@link features/readonly/README read-only mode}:
+
+```jsx
+<CKEditor
+	data="<p>Editor's content</p>"
+	readOnly={true}
+/>
+```
+
+This property takes precedence over the {@linkapi CKEDITOR.config#readOnly `config.readOnly`} setting.
+
+Defaults to `false`.
+
+### `style`
+
+`Object`
+
+Style rules set that will be applied to {@linkapi CKEDITOR.editor#container editor container} with {@linkapi CKEDITOR.dom.element#setStyles}.
+
+```jsx
+<CKEditor
+	data="<p>Editor's content</p>"
+	style={{
+		'margin-top': '100px',
+		'border': '5px solid red'
+	}}
+/>
+```
+
+Require [type](#type) to be `classic`.
+
+Defaults to `undefined`.
+
+### `type`
+
+`'classic'` | `'inline'`
+
+By default, the CKEditor 4 React component creates a {@link guide/dev/framed/README classic editor}. To create an {@link guide/dev/inline/README inline editor}, add the `type` property with the value of `inline` to the `<CKEditor />` tag:
+
+```jsx
+<CKEditor
+	data="<p>Some initial data</p>"
+	type="inline"
+/>
+```
+
+You can also explicitly set the `type` property to `classic` to create the classic editor:
+
+```jsx
+<CKEditor
+	data="<p>Some initial data</p>"
+	type="classic"
+/>
+```
+
+Every other value of the `type` property will be treated as `classic`.
+
+For more details, look at {@link guide/dev/ckeditor_js_load/README Loading CKEditor Script article}.
+
+Defaults to `'classic'`.
+
+### `onBeforeLoad`
+
+`Function`
+
+Callback function with single argument: `CKEDITOR` namespace. It is invoked **each time new editor instance is loaded**, but `CKEDITOR` object always refers to the same namespace.
```

Comment:
Let's add a note that for modifying the `CKEDITOR` namespace it is recommended to use `onNamespaceLoaded` property instead.

Reply:
I added a note.